Formula/vowpal-wabbit.rb

```diff
@@ -35,14 +35,13 @@ def install
                             "-DFMT_SYS_DEP=ON",
                             "-DSPDLOG_SYS_DEP=ON",
                             "-DVW_BOOST_MATH_SYS_DEP=On",
-                            "-DBUILD_FLATBUFFERS=ON",
                             "-DVW_INSTALL=On"
       system "make", "install"
     end
     bin.install Dir["utl/*"]
     rm bin/"active_interactor.py"
     rm bin/"vw-validate.html"
-    rm bin/"clang-format"
+    rm bin/"clang-format.sh"
```

Comment:
It may not hurt to keep this, since this likely doesn’t conflict with other formulae. But if it’s not useful deleting it is fine.

Reply:
This is just a file name change